hw/ip/spi_host/doc/_index.md

```diff
@@ -302,13 +302,16 @@ $$T_\textrm{\textrm{SCK},0}=\frac{1}{2}\frac{T_\textrm{core}}{\textrm{CONFIGOPTS
 Alternatively, this clock-divide feature can also be used to entirely bypass the need for an independent core clock.
 Instead the core can be driven by the TL-UL bus clock, and the SCK period can be adjusted using the {{< regref CONFIGOPTS_0.CLKDIV_0 >}} setting.
 
-#### Chip-select timing control
+#### Chip-select Timing Control
 
 Typically the CSB line is automatically deasserted after the last edge of SCK.
-However,  by asserting {{< regref "COMMAND.CSAAT" >}} when issuing a particular command, one can instruct the core to hold CSB low indefinitely after the last clock edge.
-This is useful for merging two adjacent commands together, to create very long commands such as continuous read operations.
+However, by asserting {{< regref "COMMAND.CSAAT" >}} when issuing a particular command, one can instruct the core to hold CSB low indefinitely after the last clock edge.
+This is useful for merging two adjacent command segments together, to create more complex commands, such as flash Quad read commands which require a mix of segments with different speeds and directions.
```

Comment:
@martin-lueker for this particular point, does it make sense in spi_host DV to test this out to make sure there arent' any got-ya's in creating these commands? 

It may make sense to have samples in the programmer's model on how we could do this.

Reply:
Absolutely agree on both points.  Are any text changes recommended here?

Reply:
no i dont think so, i think more a point for the future.